lua/lspconfig/dartls.lua

```diff
@@ -5,7 +5,10 @@ local server_name = 'dartls'
 local bin_name = 'dart'
 
 local find_dart_sdk_root_path = function()
-  if vim.fn['executable'] 'flutter' == 1 then
+  if os.getenv 'FLUTTER_SDK' ~= nil then
+    local flutter_path = os.getenv 'FLUTTER_SDK'
+    return util.path.path_join(flutter_path, '/cache/dart-sdk/bin/dart')
```

Comment:
Still needs to change

Reply:
Change how?

Reply:
`util.path.path_join(flutter_path, 'cache', 'dart-sdk', 'bin', 'dart')`